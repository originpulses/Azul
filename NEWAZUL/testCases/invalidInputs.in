this is an input
5
7
3 #valid input
9
#
j