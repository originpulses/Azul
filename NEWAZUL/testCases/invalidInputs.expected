Menu
----
1. New Game
2. Load Game
3. Credits (Show student information)
4. Quit

> Invalid Input, Enter again
> Invalid Input, Enter again
> Invalid Input, Enter again
> Invalid Input, Enter again
> Invalid Input, Enter again
> Invalid Input, Enter again
> 

Goodbye
